changelog/44.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add a new spectrogram class `radiospectra.spectrogram2.spectrogram.BaseSpectrogram` and factory `radiospectra.spectrogram2.spectrogram.SpectrogramFactory` with sources for `~radiospectra.spectrogram2.sources.SWAVESSpectrogram`, `~radiospectra.spectrogram2.sources.RFSSpectrogram`, `~radiospectra.spectrogram2.sources.CALISTOSpectrogram`, `~radiospectra.spectrogram2.sources.EOVSASpectrogram` and `~radiospectra.spectrogram2.sources.RSTNSpectrogram`.
+Add `sunpy.net.Fido` clients for `~radiospectra.net.sources.callisto.CALLISTOClient`, `~radiospectra.net.sources.eovsa.EOVSAClient` and `~radiospectra.net.sources.rstn.RSTNClient`.
```

Comment:
I would add a second feature changelog entry for this.

Reply:
Ok I'm not sure what the naming convention for that should be `44b.feature.rst`?